model/gen-ai/events.yaml

```diff
@@ -7,309 +7,32 @@ groups:
     attributes:
       - ref: gen_ai.system
 
-  - id: event.gen_ai.system.message
-    name: gen_ai.system.message
+  - id: event.gen_ai.inputs
```

Comment:
At the SIG today, we discussed supporting refs in the events too. Do you have the changes you drafted on screen?

Reply:
added event definition